libbeat/logp/syslog_unix.go

```diff
@@ -53,7 +53,7 @@ func (c *syslogCore) Write(entry zapcore.Entry, fields []zapcore.Field) error {
 	}
 
 	// Console encoder writes tabs which don't render nicely with syslog.
-	replaceTabsWithSpaces(buffer.Bytes(), 2)
+	replaceTabsWithSpaces(buffer.Bytes(), 4)
```

Comment:
Is that change related to this PR?

Reply:
It's not related. I just realized that there can be up to 4 tabs.